src/components/sidebar/lineView/lineHeaderTableRows.scss

```diff
@@ -0,0 +1,58 @@
+@import '../../shared/styles/common';
+@import '../../shared/styles/flexbox';
+@import '../../shared/styles/form';
+
+$cellPadding: 5px 2px 5px 2px; // same as in lineHeaderTable.scss
+$rowHighlightColor: $lightblue;
+
+.lineHeaderTableRow {
+    font-size: $smallFontSize;
+    border-bottom: 1px solid #dfdfdf;
+
+    .lineHeaderTableNameCell {
+        max-width: 180px;
+        padding: 5px 2px 5px 2px;
+    }
+
+    .lineHeaderTableCalendarCell {
+        min-width: 125px;
+        padding: $cellPadding;
+        vertical-align: top;
+
+        .timeInput {
+            padding-top: 0px;
+        }
+    }
+
+    .lineHeaderTableButtonCell {
+        vertical-align: top;
+        padding: $cellPadding;
+    }
+
+    .lineHeaderButton {
+        display: flex;
+        font-size: $largeFontSize;
+        padding: 5px;
+        margin-top: 4px;
+        margin-bottom: auto;
+        border-radius: 5px;
+    }
+
+    .disabledRemoveLineHeaderButton {
+        color: $disabledGrey;
+        background-color: $backgroundlightGrey;
+
+        &.highlightedBackground {
+            background-color: $rowHighlightColor;
+        }
+    }
+
+    .removeLineHeaderButton:hover {
+        color: white;
+        background-color: $red;
+    }
+}
+
+.lineHeaderRowHighlight {
```

Comment:
Could we have border-radius here and some padding on the left side of the div?

Reply:
There is already 2px padding on both sides. Increasing padding isn't a good idea since the space is already very limited.

I don't like border-radius there